test/components/search-results.test.js

```diff
@@ -12,12 +12,14 @@ import sinon from 'sinon';
 
 describe('Component', function () {
   global.dataLayer = [];
+  global.impressionsTimestamp = new Map();
```

Comment:
I understand why you would want to use a `Map` for this feature, but given the _incomplete_ support for it on Mobile devices: http://kangax.github.io/compat-table/es6/#test-Map Babel will transpile it to a "normal" Object ... right?